articles/energy-data-services/how-to-manage-data-security-and-encryption.md

```diff
@@ -0,0 +1,95 @@
+
+## Encrypt data in transit
+
+Microsoft Energy Data Services supports Transport Layer Security (TLS 1.2) protocol to protect data when it’s traveling between the cloud services and customers. TLS provides strong authentication, message privacy, and integrity (enabling detection of message tampering, interception, and forgery), interoperability, and algorithm flexibility.
+
+In addition to TLS, when you interact with Microsoft Energy Data Services, all transactions take place over HTTPS. 
+
+## Set up Customer Managed Keys (CMK) for Microsoft Energy Data Services Preview instance
+
+### Pre-requisites
+
+**Step 1- Configure the key vault**
+
+1.	You can use a new or existing key vault to store customer-managed keys. To learn more about Azure Key Vault, see [Azure Key Vault Overview](../key-vault/general/overview.md) and [What is Azure Key Vault](../key-vault/general/basic-concepts.md)?.
+2.	Using customer-managed keys with Microsoft Energy Data Services requires that both soft delete and purge protection be enabled for the key vault. Soft delete is enabled by default when you create a new key vault and cannot be disabled. You can enable purge protection either when you create the key vault or after it is created.
+3.	To learn how to create a key vault with the Azure portal, see [Quickstart: Create a key vault using the Azure portal](../key-vault/general/quick-create-portal.md). When you create the key vault, select Enable purge protection.
+
+ [![Screenshot of enabling purge protection and soft delete while creating key vault](media/data-security-and-encryption/cmek_1_create_key_vault.png)](media/data-security-and-encryption/cmek_1_create_key_vault.png#lightbox)
+ 
+4.	To enable purge protection on an existing key vault, follow these steps:
+*	Navigate to your key vault in the Azure portal.
+*	Under Settings, choose Properties.
+*	In the Purge protection section, choose Enable purge protection.
+
+**Step 2 - Add a key**
+1.	Next, add a key to the key vault.
+2.	To learn how to add a key with the Azure portal, see [Quickstart: Set and retrieve a key from Azure Key Vault using the Azure portal](../key-vault/keys/quick-create-portal.md).
+3.	It is recommended that the RSA key size is 3072, see [Configure customer-managed keys for your Azure Cosmos DB account | Microsoft Learn](../cosmos-db/how-to-setup-customer-managed-keys.md#generate-a-key-in-azure-key-vault).
+
+**Step 3 - Choose a managed identity to authorize access to the key vault**
+1.	When you enable customer-managed keys for an existing Microsoft Energy Data Services Preview instance you must specify a managed identity that will be used to authorize access to the key vault that contains the key. The managed identity must have permissions to access the key in the key vault.
```

Comment:
When you enable customer-managed keys for an existing Microsoft Energy Data Services Preview instance**,** you must specify a managed identity that will be used to authorize access to the key vault that contains the key.